tests/foreman/api/test_discoveryrule.py

```diff
@@ -0,0 +1,204 @@
+# -*- encoding: utf-8 -*-
+"""API Tests for foreman discovery feature"""
+from ddt import ddt
+from fauxfactory import gen_string, gen_integer, gen_choice
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.common.decorators import data, run_only_on
+from robottelo.test import APITestCase
+
+
+def _discoveryrule_names():
```

Comment:
Do we have some kind of requirement to have length of string equal 15? If not, could you please remove that method entirely and use valid_data_list() from helpers.py instead?